src/pathways/escher.service.ts

```diff
@@ -1,100 +1,135 @@
 import * as escher from 'escher-vis'
 import * as d3 from 'd3';
+import * as _ from 'lodash';
+import * as fp from 'lodash/fp';
 import 'jquery';
 
+/**
+ * Plan: the current structure is the following:
```

Comment:
Is this a TODO? Perhaps it should be marked as such as `/**` blocks are usually reserved for documentation. So perhaps split it into how it is currently (documentation) and then add the TODO separately? Or is this something that you want to address in this PR?